### ISSUE TITLE
Missing OGC reference metadata

### ISSUE THREAD

**Issue by indeOWS** [GeoNode/geonode-project#189]
I have been unable to add layer through the create map button. The same issue as [7306](https://github.com/GeoNode/geonode/issues/7306). I realized that the same issue occurs in GeoNode Master Demo. See layer [pontes](https://master.demo.geonode.org/layers/geonode_master_data:geonode:pontes). I wonder if the problem is related to any GeoNode configuration or the data itself?

**Comment by indeOWS** [comment]
I ran a test by uploading all the layers in GeoNode-Project@3.1.x. And create map (add layers) works like a charm. There is no issue regarding Missing OGC reference metadata. So, my first guess is that the issue is not related to the data itself, but with GeoNode-Project@3.2.x

**Comment by giohappy** [comment]
@indeOWS it's not clear what's the problem. Are you trying to load a layer from the MapStore client CSW catalogue? Are you trying to create a map from an exising layer?

**Comment by indeOWS** [comment]
@giohappy, similar issue as in issue [7306](https://github.com/GeoNode/geonode/issues/7306). I uploaded a shapefile [pontes](https://master.demo.geonode.org/layers/geonode_master_data:geonode:pontes)  in GeoNode Master Demo. The layer reports "Missing OGC reference metadata". And If you try to create a map with this layer, the plus sign (+) to add a layer won't show up. This is happening with all layers I am uploading in GeoNode-Project 3.2.x, from both methods upload layer and manage.sh updatelayers. I ran a test on GeoNode-Project@3.1.x, and 3.1.x works like a charm, no problem at all. There is no message <Missing OGC reference metadata> on 3.1.x version. And the + sign is on. Find attached the pontes.shp

**Steps to Reproduce the Problem**

Upload an layer to GeoNode
Select Create Map
Select Add layer

**Specifications**

GeoNode version:3.2.x
Installation method (GeoNode Docker)
Platform: Ubuntu 20.04 LTS Server
[ponte.zip](https://github.com/GeoNode/geonode-project/files/6599330/ponte.zip)
